src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -112,6 +112,25 @@ private sealed class SetOnFinalized
     public sealed class SocketsHttpHandler_HttpProtocolTests : HttpProtocolTests
     {
         public SocketsHttpHandler_HttpProtocolTests(ITestOutputHelper output) : base(output) { }
+
+        [Fact]
+        public async Task DefaultRequestHeaders_Invalid_SentUnparsed()
```

Comment:
Technically, the header isn't invalid right? It's just not parsed/validated.

Reply:
Oops, that's leftover from when I was using one that was invalid.  Will fix the name.

Reply:
Might be worthwhile to test both -- valid but not parsed, and also invalid (and still sent).